app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -839,6 +877,8 @@ html.omni-nav-v2 {
   }
 }
 
+table.gsc-completion-container { font-size: 13px; }
+
```

Comment:
I tried nesting this in the `omni-nav-v2` selector but this element sits outside that element and not as a sibling. This was the only way I could find a way to select the font size. 
![screen shot 2018-04-10 at 12 01 18 pm](https://user-images.githubusercontent.com/29960336/38579698-a644c760-3cbc-11e8-91a5-fecba525f248.png)